.travis.yml

```diff
@@ -47,7 +47,7 @@ jobs:
     include:
     -   stage: Tests
         name: "Smoke, Functional and Acceptance Tests"
-        script: docker-compose exec php-fpm ./phing db-create test-db-create build-demo-dev-quick error-pages-generate checks-ci
+        script: docker-compose exec php-fpm ./phing db-create test-db-create build-demo-dev-quick monorepo-backend-api-install backend-api-oauth-keys-generate error-pages-generate checks-ci
 
     -   name: "Standards, Unit Tests"
         script: docker-compose exec php-fpm ./phing standards tests-unit
```

Comment:
maybe also here .. `monorepo-backend-api-install backend-api-oauth-keys-generate`

Reply:
sorry, what should I do? I don't understand :innocent:

Reply:
```yaml
        script: docker-compose exec php-fpm ./phing monorepo-backend-api-install backend-api-oauth-keys-generate standards tests-unit
```